scripts/cr_auto_iterate.py

```
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+import json
+import os
+import re
```

Ruff (0.12.2)

`re` imported but unused

Remove unused import: `re`

(F401)